webapp/channels/src/components/admin_console/access_control/editors/table_editor/table_editor.scss

```diff
@@ -1,110 +1,45 @@
 .table-editor {
-    position: relative;
     margin-bottom: 24px;
 
     &__table {
-        overflow: hidden;
+        width: 100%;
         border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
         border-radius: 4px;
+        border-collapse: collapse;
     }
 
-    &__header {
-        display: flex;
-        padding: 12px 16px;
+    th {
+        padding: 12px;
         border-bottom: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
         background: rgba(var(--center-channel-color-rgb), 0.04);
-    }
-
-    &__column-header {
-        flex: 1;
-        color: var(--center-channel-color);
-        font-size: 14px;
         font-weight: 600;
-        padding-inline: 10px;
-
-        &:nth-child(1) {
-            flex: 1;
-        }
-
-        &:nth-child(2) {
-            flex: 0.8;
-        }
-
-        &:nth-child(3) {
-            flex: 2.3;
-        }
+        text-align: left;
     }
 
-    &__column-header-actions {
-        color: var(--center-channel-color);
-        font-size: 14px;
-        font-weight: 600;
-    }
-
-    &__row {
-        display: flex;
-        align-items: center;
-        padding: 0;
+    td {
         border-bottom: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+        vertical-align: middle;
     }
 
-    &__cell {
-        flex: 1;
-
-        &:nth-child(1) {
-            flex: 1;
-        }
-
-        &:nth-child(2) {
-            flex: 0.8;
-        }
-
-        &:nth-child(3) {
-            flex: 2;
-        }
-    }
-
-    &__cell-actions {
-        width: 40px;
-        text-align: right;
+    .table-editor__column-header-value {
+        padding-inline: 10px;
     }
 
-    &__attribute-select,
-    &__operator-select {
-        width: 100%;
-    }
+    // Set column widths
+    th:nth-child(2), td:nth-child(2) { width: 20%; }
+    th:nth-child(3), td:nth-child(3) { width: 50%; }
+    th:nth-child(4), td:nth-child(4) { width: 30px; text-align: right; }
 
-    &__select {
-        width: 100%;
-        padding: 10px 16px;
-        border: none;
-        border-radius: 4px;
-        appearance: none;
-        background-color: transparent;
-        color: var(--center-channel-color);
-        font-size: 14px;
-        
-        &:hover {
-            background-color: rgba(var(--button-bg-rgb), 0.08);
-            cursor: pointer;
-        }
+    &__blank-state {
+        padding: 16px;
+        text-align: center;
         
-        &:focus {
-            background-color: rgba(var(--button-bg-rgb), 0.08);
-            outline: none;
-        }
-
-        &:disabled {
-            cursor: not-allowed;
-            opacity: 0.6;
+        span {
+            color: rgba(var(--center-channel-color-rgb), 0.64);
```

Comment:
> There's a few more places in this file, cel_editor/editor.scss, and job_details_modal.scss that use the deprecated versions of the CSS variables. Could you change those to use the rgba version as above?
>
> I'll also go make a ticket to look at actually removing those since it seems like they've been almost entirely eliminated from the code except a few places they've been added back in just recently

https://github.com/mattermost/mattermost/pull/30785#discussion_r2089074469 comment from @hmhealey 

Updated.